backend/src/main/java/gov/cdc/usds/simplereport/db/model/Person.java

```diff
@@ -75,7 +75,7 @@ public class Person extends OrganizationScopedEternalEntity implements PersonEnt
    * Note that for the purposes of all upserts, the <em>first</em> phone number in a
    * List<PhoneNumber> is considered to be the primary
    */
-  @OneToOne(fetch = FetchType.EAGER)
+  @OneToOne(fetch = FetchType.LAZY)
```

Comment:
Setting this back to lazy since I added the dataloader, and there is no need to always load it if its not used